packages/connect-explorer/src/js/components/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const HeaderComponent = styled.header`
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 50px;
+    background: #060606;
+    color: #f6f7f8;
+    overflow: hidden;
+    min-width: 780px;
+    z-index: 100;
+`;
+
+const Svg = styled.svg`
+    fill: #ffffff;
+    height: 28px;
+    width: 100px;
+    margin-top: 9px;
+    display: inline-block;
+`;
+
+const Title = styled.span`
+    display: inline-block;
+    vertical-align: top;
+    margin-top: 16px;
+    margin-left: 20px;
+`;
+
+// todo
+const LayoutWrapper = styled.div``;
```

Comment:
Why not adding just a `div` for now and when we need we can easily create `LayoutWrapper` styled ?

Reply:
removed